db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True, null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255, unique=True)
+    bio = models.CharField(max_length=255, blank=True)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
```

Comment:
Consider whether `on_delete=models.CASCADE` is appropriate here. Use `on_delete=models.CASCADE` if the Player should be deleted when the associated Race is deleted. Otherwise, consider using `on_delete=models.SET_NULL` if the Player should remain even if the Race is deleted.